<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TranDucTrungKien</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/siu.gif</normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/siu.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>Tinh Dum Tao</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelinputN">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input N:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditN">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrimeNumber">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prime Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Even Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrimeNumber">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEvenNumber">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>341</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Odd Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOddNumber">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOk">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
